lib/class-wp-rest-widget-updater-controller.php

```diff
@@ -76,53 +75,113 @@ public function compute_new_widget_permissions_check() {
 	}
 
 	/**
-	 * Returns the new widget instance and the form that represents it.
+	 * Checks if the widget being referenced is valid.
 	 *
 	 * @since 5.2.0
-	 * @access public
+	 * @param string  $identifier         Widget id for callback widgets or widget class name for class widgets.
+	 * @param boolean $is_callback_widget If true the widget is a back widget if false the widget is a class widget.
 	 *
-	 * @param WP_REST_Request $request Full details about the request.
-	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 * @return boolean True if widget being referenced exists and false otherwise.
 	 */
-	public function compute_new_widget( $request ) {
-		$url_params = $request->get_url_params();
-
-		$widget = $request->get_param( 'identifier' );
-
+	private function is_valid_widget( $identifier, $is_callback_widget ) {
+		if ( null === $identifier ) {
```

Comment:
It seems like we should just be making these `args` as `required` ? Rather than trying to deal with validating the request format from the implementation.